src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/impl/ActionSvcRestClientImpl.java

```diff
@@ -225,13 +225,14 @@ public ActionRuleDTO updateActionRule(
             appConfig.getActionSvc().getActionRulesForActionPlanPath(), null, actionPlanId);
 
     ResponseEntity<List<ActionRuleDTO>> response;
+    HttpEntity httpEntity = restUtility.createHttpEntity(null);
```

Comment:
This is the line that adds the authentication header?

Reply:
It's not very obvious... I've create https://github.com/ONSdigital/rm-common-service/pull/14/files to help